package.json

```diff
@@ -26,6 +26,7 @@
     "webpack-cli": "^4.9.1"
   },
```

Comment:
- [ ] On line 6, the _script option_ of your _package.json_ file is **not properly configured** and hence your application cannot start using the command `npm start`. Kindly add the necessary _script options_ similar to as can be seen below:

```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```